src/utils.js

```diff
@@ -40,3 +40,7 @@ export const getColorContrast = (relativeLuminance1, relativeLuminance2) => {
   const contrast = (lighterColor + 0.05) / (darkerColor + 0.05)
   return contrast
 }
+
+export const deepCopyArrayOfObject =(array)=>{
+  return array.map(a => ({...a}))
+}
```

Comment:
Please add an endline

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
you mean terminator(;)?

Reply:
An endline means press enter so that the file finishes on line 47 instead of line 46